src/Google/Ads/GoogleAds/Constants.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/**
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Ads\GoogleAds;
+
+/**
+ * Class to store constants and default values of the library.
+ */
+class Constants
+{
+    // The library name to use in Google's request headers.
+    public const NAME = 'gccl';
```

Comment:
Can we use LIBRARY_NAME here? It's not clear from when it's used (`Constants::NAME`) that this is a library name.

Reply:
Good point. I did change `VERSION` as well.